runners/wikiConsole/templates/createDevon4ngProject.asciidoc

```diff
@@ -0,0 +1,15 @@
+== Create devon4ng project
+
+<% if (cdCommand) { %>
+
+=== Prerequisites
+
+As the project should be created at `<%= cdCommand; %>`, make sure that this directory exists before proceeding with the next step. 
+
+<% } %>
+
+=== Creating the Angular project
+
+Use the `ng` command provided by the `devonfw-ide` to create the project <%= projectName; %>:
```

Comment:
We have used devonfw IDE in all the other templates.
I think, that this function should work without devonfw too. 
At least we have implemented the option in all the other runners.